Formula/conjure-up.rb

```diff
@@ -107,7 +107,7 @@ class ConjureUp < Formula
     # Python 3.8 compatibility platform.linux_distribution
     # Remove in next release
     patch do
-      url "https://github.com/go-macaroon-bakery/py-macaroon-bakery/pull/75.patch?full_index=1"
+      url "https://github.com/go-macaroon-bakery/py-macaroon-bakery/commit/78daf9d233e33da3f4bd2c34553843f82c09b21e.patch?full_index=1"
```

Comment:
Is there a preference for `.patch` or `.diff`?

I've been using `.diff` mostly because I don't think we need the extra metadata that comes with a patch, but I doubt it makes much of a difference.

I used `.patch` here to avoid unnecessary checksum changes and to get a cleaner diff.